nuspec/Forms/CoreContent/CoreApp.cs.pp

```diff
@@ -12,7 +12,7 @@
                 .AsInterfaces()
                 .RegisterAsLazySingleton();
 
-            RegisterAppStart<ViewModels.MainViewModel>();
+            RegisterAppStart<ViewModels.MvxFormsViewModel>();
```

Comment:
We changed from some other name to `MainViewModel` for consistency with the other StarterPacks, so I don't see why we'd want to change it back.

Reply:
I agree, but if your ViewModel is called MainViewModel you will get the error message:  Problem seen creating View-ViewModel lookup table - you have more than one View registered for the ViewModels: 2*MainViewModel (MainActivity,MvxFormsPage). Or we have to do it like this or we have to change the name of MainActivity so something like FormsActivity (also what @martijn00 suggests in his blog on Medium)

Reply:
I would vote for keeping the name `MainViewModel` and `MainPage` in Forms, but rename `MainActivity` to `FormsActivity`. This is also because in the normal startpack we add a MainView and tell people to remove old activities, like MainActivity.

Reply:
Changed it to 'MainViewModel' and 'MainPage'. Removing the MainActivity from the Droid project is also in the ToDo file now.